src/google_map.js

```diff
@@ -502,7 +506,9 @@ export default class GoogleMap extends Component {
           return;
         }
 
-        const centerLatLng = this.geoService_.getCenter();
+        const centerLatLng = this.pendingProps_
+          ? latLng2Obj(this.pendingProps_.center)
+          : this.geoService_.getCenter();
```

Comment:
Your assign pendingProps_ but never "unassign" it (on line 379). `this.geoService_.getCenter();` will never be called after the map was initialized ( `this.map_ == true`). It's look's like weird, no?